src/components/Search.js

```diff
@@ -1,9 +1,47 @@
-import React from 'react'
+import React, { Component } from 'react'
+import { searchMonster } from '../actions/actions'
+import { connect } from 'react-redux'
+import './Search.css'
 
-const Search = () => {
+class Search extends Component {
+  constructor() {
```

Comment:
Wondering if we can implement hooks to whittle these class components down - and we could memoize searches/monsters that have already been searched, to cut down on API calls.